Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/Resources/CaptureDataTargetsFileTemplate.xml

```diff
@@ -6,6 +6,7 @@
     <Message Importance='high' Text='CAPTURE::PROPERTY::TargetFileName::$(TargetFileName)' />
     <Message Importance='high' Text='CAPTURE::PROPERTY::ErrorLog::$(ErrorLog)' />
     <Message Importance='high' Text='CAPTURE::PROPERTY::SonarProjectOutFolderFilePath::$(SonarProjectOutFolderFilePath)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::SonarProjectConfigFilePath::$(SonarProjectConfigFilePath)' />
```

Comment:
could you add this file and the other `Resources` in the csproj, so that VS can see it?

Reply:
No, they are intentionally in embedded project resources to avoid access hell. 
Double click "Properties" in VS/Solution explorer, they are accessible through `Resources` tab.